src/dateTime.js

```diff
@@ -3,17 +3,23 @@ function getDateTime() {
   var month = parseInt(dateObj.getMonth() + 1);
   var day = parseInt(String(dateObj.getDate() + 1).padStart(2, '0'));
   var year = parseInt(dateObj.getFullYear());
+  // hour is like this coz getHours() is a bit whack, shows 2300 when it is 11am and 1100 when it is 11pm
   var hour = (parseInt(dateObj.getHours()) + 1 > 12) 
```

Comment:
.getHours() worked correctly previously :/
Issue is probably your test bot is on New York time hehe.
Commit to this same pr